cfme/containers/provider/openshift.py

```diff
@@ -137,6 +158,16 @@ def __init__(
             endpoints=endpoints,
             appliance=appliance)
 
+    def create(self, **kwargs):
+
+        # Enable alerts collection before adding the provider to avoid missing active
+        # alert after adding the provider
+        # For more info: https://bugzilla.redhat.com/show_bug.cgi?id=1514950
+        if getattr(self, "alerts_type", None) == "Prometheus":
+            self.enable_all_alerts_collection()
```

Comment:
Discussed this offline, wanted to record comment here.

I think this is ultimately driving a bit of an anti-pattern, in an effort to work around a bug, and is resulting in a bunch of layered convenience methods that make a lot of assumptions, and repeat the functionality provided by `BaseAlertProfile.assign_to()`

I think the new methods should be collapsed into a few lines here instantiating the alert profiles for node and provider, and then using their `assign_to()` methods, passing 'The Enterprise' during provider configuration.

Consider wrapping this behavior in an `if BZ(1514950).blocks()` block, so that we stop doing this workaround as soon as the bug is fixed.